<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Input</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="inputEdit">
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="outputEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>160</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="inputButton">
       <property name="text">
        <string>Записать в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="outputButton">
       <property name="text">
        <string>Считать из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>Очистить inputEdit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>451</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите текст, который будет записан в файл</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Добавьте элементы, которые будут записаны в файл используя Enter </string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="maximum">
       <double>2147483223454.989990234375000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_10_Add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Модель ноутбука</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7_Add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Количество USB портов</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4_Add">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>158</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_Add">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>7</y>
        <width>48</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="CheckBox_Add">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>99</y>
        <width>21</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_8_Add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Частота ядра процессора</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2_Add">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>68</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9_Add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Является ли игровым</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11_Add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID ноутбука</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Add">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>38</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3_Add">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>128</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12_Add">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Название компании</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13_Add">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>401</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QPushButton" name="addButtonBin">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7_Add_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Количество USB портов</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11_Add_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>133</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID ноутбука</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8_Add_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>43</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Частота ядра процессора</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4_Add_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>161</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9_Add_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>103</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Является ли игровым</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Add_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>41</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="CheckBox_Add_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>102</y>
        <width>21</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13_Add_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>381</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2_Add_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>71</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12_Add_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>163</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Название компании</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3_Add_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>131</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10_Add_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>73</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Модель ноутбука</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_Add_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>48</width>
        <height>29</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>451</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Введите текст, который будет дозаписан в файл</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>371</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Запись/чтение текста в файл</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Запись/чтение массива типа float</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дозапись/чтение полей структуры Notebook в txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дозапись/чтение полей структуры Notebook в bin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дозапись/чтение текста в файл</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup_4"/>
 </buttongroups>
</ui>
